### PR TITLE
Feat/hh test coverage

### PR DESCRIPTION
**Pull request by Aboudjem**
- +10% hh coverage

### CODE REVIEW

**Review by ankurdubey521 — APPROVED** [review]

File: contracts/Nexus.sol [hunk 1]
```diff
@@ -313,7 +313,7 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
     /// as Biconomy v2 Account (proxy) reads implementation from the slot that is defined by its address
     /// @param newImplementation The address of the new contract implementation.
     /// @param data The calldata to be sent to the new implementation.
-    function upgradeToAndCall(address newImplementation, bytes calldata data) public payable virtual override onlyEntryPointOrSelf withHook {
```
Comment (livingrockrises) [inline contracts/Nexus.sol:316]: why is this redundant?

**Review by livingrockrises — COMMENTED** [review]
review i

File: contracts/Nexus.sol (hunk 1 above)
Comment (Aboudjem) [reply contracts/Nexus.sol:316]: it's already on `_authorizeUpgrade`

Comment (Aboudjem) [reply contracts/Nexus.sol:316]: `    function _authorizeUpgrade(address newImplementation) internal virtual override(UUPSUpgradeable) onlyEntryPointOrSelf { }`

Comment (livingrockrises) [reply contracts/Nexus.sol:316]: remember discussing this with Konrad/Jon. 
I think we can remove from _authorizeUpgrade and bring back in public method

Comment (Aboudjem) [reply contracts/Nexus.sol:316]: I decided to do it this way to comply with Solady's recommendation

```

    /// @dev Please override this function to check if `msg.sender` is authorized
    /// to upgrade the proxy to `newImplementation`, reverting if not.
    /// ```
    ///     function _authorizeUpgrade(address) internal override onlyOwner {}
    /// ```
    function _authorizeUpgrade(address newImplementation) internal virtual;

```

Comment (livingrockrises) [reply contracts/Nexus.sol:316]: hmm had done this way before. but to think of it internal methods are not touched so naturally makes sense to apply on public main method. 
if needed, get more opinions on this from our team and rhinestone. 
also remember this code is already audited and will cause changed to be added in requested new review.

Comment (livingrockrises) [reply contracts/Nexus.sol:316]: ok reverting this cause dont want to confuse spearbit.

**Review by livingrockrises — COMMENTED** [review]
can hold off merge until end of spearbit review period.

File: contracts/factory/RegistryFactory.sol [hunk 2]
```diff
@@ -170,6 +170,11 @@ contract RegistryFactory is Stakeable, INexusFactory {
         expectedAddress = payable(LibClone.predictDeterministicAddressERC1967(ACCOUNT_IMPLEMENTATION, actualSalt, address(this)));
     }
 
+    function getAttesters() public view returns (address[] memory) {
```
Comment (livingrockrises) [inline contracts/factory/RegistryFactory.sol:173]: this must be acceptable for audit new scope pov. so not asking them

**Review by livingrockrises — APPROVED** [review]

**Comment by livingrockrises** [comment]
please check failing CI checks

**Pull request merged**